tests/scripts/all.sh

```diff
@@ -964,6 +964,25 @@ component_test_everest () {
     if_build_succeeded tests/compat.sh -f ECDH -V NO -e 'ARCFOUR\|ARIA\|CAMELLIA\|CHACHA\|DES\|RC4'
 }
 
+component_test_everest_curve25519_only () {
+    msg "build: Everest ECDH context, only Curve25519" # ~ 6 min
+    scripts/config.py unset MBEDTLS_ECDH_LEGACY_CONTEXT
+    scripts/config.py set MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED
+    scripts/config.py unset MBEDTLS_ECDSA_C
```

Comment:
Why do you need to unset MBEDTLS_ECDSA_C, MBEDTLS_KEY_EXCHANGE_ECDH_ECDSA_ENABLED and MBEDTLS_KEY_EXCHANGE_ECDH_ECDSA_ENABLED?

Reply:
Because you can't enable ECDSA if there isn't a curve that supports ECDSA, and you can't use an ECDSA ciphersuite if ECDSA isn't available.